source/components/definition-list/definition-list.less

```diff
@@ -0,0 +1,48 @@
+@import (reference) 'grid/grid';
+hx-dl {
+  display: block;
+  font-size: 1rem; // 16px
+
+  hx-def {
+    display: flex;
+
+    hx-dt {
+      #grid.span(2);
+      color: @gray-750;
+      display: block;
+      flex-shrink: 1;
+      font-weight: 400;
+      text-align: right;
+    }
+
+    hx-dd {
```

Comment:
Please reduce nesting. These styles should not be reliant upon their hierarchy unless the `.hxVertical` modifier is present.

```less
hx-dl { ... }
hx-def {
  ...

  & + & { ... }
}
hx-dt { ... }
hx-dd { ... }

hx-dl.hxVertical {
  hx-def { ... }
  hx-dt { ... }
  hx-dd { ... }
}